.github/workflows/discover-lean-pr-testing.yml

```diff
@@ -230,17 +230,17 @@ jobs:
           MESSAGE+=$'### ❌ Failed merges:\n\nThe following branches need to be merged manually:\n\n'
           for MERGE_INFO in $FAILED_MERGES; do
             IFS='|' read -r PR_NUMBER GITHUB_DIFF BRANCH <<< "$MERGE_INFO"
-            MESSAGE+=$(printf 'Diff: %s\n\n' "$GITHUB_DIFF")
+            MESSAGE+=$(printf 'Diff: %s' "$GITHUB_DIFF")$'\n\n'
             MESSAGE+=$'```bash\n'
-            MESSAGE+=$(printf 'scripts/merge-lean-testing-pr.sh %s\n' "$PR_NUMBER")
+            MESSAGE+=$(printf 'scripts/merge-lean-testing-pr.sh %s' "$PR_NUMBER")$'\n'
             MESSAGE+=$'```\n\n'
           done
         else
           MESSAGE+=$'All branches were successfully merged! 🎉\n'
         fi
 
         # Output the message using the correct GitHub Actions syntax
-        printf 'msg<<EOF\n%s\nEOF' "$MESSAGE" >> "$GITHUB_OUTPUT"
+        printf 'msg<<EOF\n%s\nEOF' "$MESSAGE" | tee "$GITHUB_OUTPUT"
```

Comment:
In case the `| tee` makes you queasy, I copied it from the [maintainer merge](https://github.com/leanprover-community/mathlib4/blob/f167e8ddbde220d1a70abd905821e6bf83828935/.github/workflows/maintainer_merge.yml#L88) action that works and has line breaks.